cli/src/main/scala/scalaxb/compiler/Config.scala

```diff
@@ -147,4 +150,25 @@ object ConfigEntry {
   case object AutoPackages extends ConfigEntry
   case object GenerateMutable extends ConfigEntry
   case object GenerateVisitor extends ConfigEntry
+  case object CapitalizeWords extends ConfigEntry
+
+  object SymbolEncoding {
+    sealed abstract class Strategy(val alias: String, val description: String) extends ConfigEntry with Product with Serializable {
+      final override def name: String = classOf[Strategy].getName
+    }
+    case object Discard      extends Strategy("discard",       "Discards any characters that are invalid in Scala identifiers, such as dots and hyphens")
+    case object SymbolName   extends Strategy("symbol-name",   "Replaces `.`, `-`, `:`, and trailing `_` in class names with `Dot`, `Hyphen`, `Colon`, and `Underscore`")
+    case object UnicodePoint extends Strategy("unicode-point", "Replaces symbols with a 'u' followed by the 4-digit hexadecimal code of the character (e.g. `_` => `u005f`)")
+    case object DecimalAscii extends Strategy("decimal-ascii", "Replaces symbols with a 'u' followed by the decimal code of the character (e.g. `_` => `u95`)")
```

Comment:
Nice. Thanks so much for these compat flags ❤️

Reply:
You're welcome. Hopefully, these flags will make it easier to change the default later.